README.md

```diff
@@ -166,6 +166,8 @@ const code = unMask("ABC-1234") // return ABC1234
 
 You can see an example app with Expo CLI [here](example/App.js).
 
+You can see a SignUp example working with iOS, Mobile, and Web [here](https://snack.expo.dev/@brenont/react-native-mask-text-example).
```

Comment:
Maybe is more interesting to indicate that the example is available on Snack.

```suggestion
You can see a SignUp example on Expo Snack working with iOS, Mobile, and Web [here](https://snack.expo.dev/@brenont/react-native-mask-text-example).
```

Reply:
That’s good